File: Data-Types/1. Character/2. Identification/Character Identification.py
import msvcrt

ch=raw_input("\n Enter a character : ")

n=ord(ch)
if n>=48 and n<=57:
    print "\n '"+ch+"' is a numeral digit. \n"
elif n>=65 and ch<=90:
    print "\n '"+ch+"' is an uppercase alphabet. \n"
elif n>=97 and n<=122:
    print "\n '"+ch+"' is a lowercase alphabet. \n"
else:
    print "\n '"+ch+"' is a special character. \n"
char=msvcrt.getch()

File: Data-Types/1. Character/2. Identification/Character Identification.cpp
#include<iostream>
#include<conio.h>

using namespace std;

int main()
{
	char ch;
	cout<<"\n Enter a character : ";
	ch=getche();

	if(ch>=48&&ch<=57)
	 cout<<"\n\n '"<<ch<<"' is a numeral digit. \n";
	else if(ch>=65&&ch<=90)
	 cout<<"\n\n '"<<ch<<"' is an uppercase alphabet. \n";
	else if(ch>=97&&ch<=122)
	 cout<<"\n\n '"<<ch<<"' is a lowercase alphabet. \n";
	else
	 cout<<"\n\n '"<<ch<<"' is a special character. \n";
	getch();
}
